e2e/libs/uninstallation/uninstallation.py

```diff
@@ -0,0 +1,57 @@
+from utility.utility import logging
+from node import Node
+from node_exec import NodeExec
+from k8s import k8s
+from utility.constant import LONGHORN_NAMESPACE
+from utility.constant import LONGHORN_UNINSTALL_JOB_LABEL
+from kubernetes import client
+
+import os
+
+class Uninstallation:
```

Comment:
The `Uninstallation` class is too specific. We can create a `Longhorn` abstract class includes `install`, `uninstall`, `upgrade` and other related interfaces. Then, classes like `LonghornKubectl`, `LonghornHelmChart`, `LonghornRancherChart` can extend the `Longhorn` class to implement these interfaces.

Reply:
Updated, we already have `e2e/libs/longhorn.py` so I add below in `e2e/libs/longhorn_deploy`
.
├── base.py
├── __init__.py
├── longhorn_deploy.py
├── longhorn_helm_chart.py
└── longhorn_kubectl.py